esmvaltool/recipes/testing/recipe_deangelis15nat_fig1_fast.yml

```diff
@@ -51,12 +51,11 @@ diagnostics:
       - {dataset: GFDL-CM3, exp: rcp85, start_year: 2091,
          end_year: 2091}
       - {dataset: GFDL-CM3, exp: abrupt4xCO2, start_year: 141, end_year: 141}
-      - {dataset: ACCESS1-0, exp: piControl, start_year: 300, end_year: 300}
-      - {dataset: ACCESS1-0, exp: rcp85, start_year: 2091, end_year: 2091}
-      - {dataset: ACCESS1-0, exp: abrupt4xCO2, start_year: 300, end_year: 300}
+      # This dataset requires about 7GB of data
```

Comment:
@katjaweigel Would it be possible to use a different dataset here? The recipe only uses one year of data, but this model stores many years per file so this results in having to download 7 GB of data:
```
192M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/abrupt4xCO2/mon/atmos/Amon/r1i1p1/v20120727/evspsbl_Amon_ACCESS1-0_abrupt4xCO2_r1i1p1_030001-044912.nc
192M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/abrupt4xCO2/mon/atmos/Amon/r1i1p1/v20120727/hfls_Amon_ACCESS1-0_abrupt4xCO2_r1i1p1_030001-044912.nc
192M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/abrupt4xCO2/mon/atmos/Amon/r1i1p1/v20120727/hfss_Amon_ACCESS1-0_abrupt4xCO2_r1i1p1_030001-044912.nc
192M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/abrupt4xCO2/mon/atmos/Amon/r1i1p1/v20120727/pr_Amon_ACCESS1-0_abrupt4xCO2_r1i1p1_030001-044912.nc
192M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/abrupt4xCO2/mon/atmos/Amon/r1i1p1/v20120727/rlds_Amon_ACCESS1-0_abrupt4xCO2_r1i1p1_030001-044912.nc
192M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/abrupt4xCO2/mon/atmos/Amon/r1i1p1/v20120727/rlus_Amon_ACCESS1-0_abrupt4xCO2_r1i1p1_030001-044912.nc
192M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/abrupt4xCO2/mon/atmos/Amon/r1i1p1/v20120727/rlut_Amon_ACCESS1-0_abrupt4xCO2_r1i1p1_030001-044912.nc
192M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/abrupt4xCO2/mon/atmos/Amon/r1i1p1/v20120727/rsds_Amon_ACCESS1-0_abrupt4xCO2_r1i1p1_030001-044912.nc
192M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/abrupt4xCO2/mon/atmos/Amon/r1i1p1/v20120727/rsdt_Amon_ACCESS1-0_abrupt4xCO2_r1i1p1_030001-044912.nc
192M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/abrupt4xCO2/mon/atmos/Amon/r1i1p1/v20120727/rsus_Amon_ACCESS1-0_abrupt4xCO2_r1i1p1_030001-044912.nc
192M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/abrupt4xCO2/mon/atmos/Amon/r1i1p1/v20120727/rsut_Amon_ACCESS1-0_abrupt4xCO2_r1i1p1_030001-044912.nc
319M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/piControl/mon/atmos/Amon/r1i1p1/v20130524/evspsbl_Amon_ACCESS1-0_piControl_r1i1p1_030001-054912.nc
319M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/piControl/mon/atmos/Amon/r1i1p1/v20130524/hfls_Amon_ACCESS1-0_piControl_r1i1p1_030001-054912.nc
319M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/piControl/mon/atmos/Amon/r1i1p1/v20130524/hfss_Amon_ACCESS1-0_piControl_r1i1p1_030001-054912.nc
319M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/piControl/mon/atmos/Amon/r1i1p1/v20130524/pr_Amon_ACCESS1-0_piControl_r1i1p1_030001-054912.nc
319M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/piControl/mon/atmos/Amon/r1i1p1/v20130524/rlds_Amon_ACCESS1-0_piControl_r1i1p1_030001-054912.nc
319M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/piControl/mon/atmos/Amon/r1i1p1/v20130524/rlus_Amon_ACCESS1-0_piControl_r1i1p1_030001-054912.nc
319M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/piControl/mon/atmos/Amon/r1i1p1/v20130524/rlut_Amon_ACCESS1-0_piControl_r1i1p1_030001-054912.nc
319M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/piControl/mon/atmos/Amon/r1i1p1/v20130524/rsds_Amon_ACCESS1-0_piControl_r1i1p1_030001-054912.nc
319M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/piControl/mon/atmos/Amon/r1i1p1/v20130524/rsdt_Amon_ACCESS1-0_piControl_r1i1p1_030001-054912.nc
319M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/piControl/mon/atmos/Amon/r1i1p1/v20130524/rsus_Amon_ACCESS1-0_piControl_r1i1p1_030001-054912.nc
319M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/piControl/mon/atmos/Amon/r1i1p1/v20130524/rsut_Amon_ACCESS1-0_piControl_r1i1p1_030001-054912.nc
122M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/rcp85/mon/atmos/Amon/r1i1p1/v20120727/evspsbl_Amon_ACCESS1-0_rcp85_r1i1p1_200601-210012.nc
122M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/rcp85/mon/atmos/Amon/r1i1p1/v20120727/hfls_Amon_ACCESS1-0_rcp85_r1i1p1_200601-210012.nc
122M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/rcp85/mon/atmos/Amon/r1i1p1/v20120727/hfss_Amon_ACCESS1-0_rcp85_r1i1p1_200601-210012.nc
122M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/rcp85/mon/atmos/Amon/r1i1p1/v20120727/pr_Amon_ACCESS1-0_rcp85_r1i1p1_200601-210012.nc
122M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/rcp85/mon/atmos/Amon/r1i1p1/v20120727/rlds_Amon_ACCESS1-0_rcp85_r1i1p1_200601-210012.nc
122M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/rcp85/mon/atmos/Amon/r1i1p1/v20120727/rlus_Amon_ACCESS1-0_rcp85_r1i1p1_200601-210012.nc
122M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/rcp85/mon/atmos/Amon/r1i1p1/v20120727/rlut_Amon_ACCESS1-0_rcp85_r1i1p1_200601-210012.nc
122M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/rcp85/mon/atmos/Amon/r1i1p1/v20120727/rsds_Amon_ACCESS1-0_rcp85_r1i1p1_200601-210012.nc
122M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/rcp85/mon/atmos/Amon/r1i1p1/v20120727/rsdt_Amon_ACCESS1-0_rcp85_r1i1p1_200601-210012.nc
122M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/rcp85/mon/atmos/Amon/r1i1p1/v20120727/rsus_Amon_ACCESS1-0_rcp85_r1i1p1_200601-210012.nc
122M	/home/bandela/climate_data/cmip5/output1/CSIRO-BOM/ACCESS1-0/rcp85/mon/atmos/Amon/r1i1p1/v20120727/rsut_Amon_ACCESS1-0_rcp85_r1i1p1_200601-210012.nc
6,8G	total
```